src/lightbox/LightboxContainer.js

```diff
@@ -71,7 +71,7 @@ class LightboxContainer extends PureComponent<Props, State> {
       buttonIndex => {
         executeActionSheetAction({
           title: options[buttonIndex],
-          src,
+          src: src.uri,
```

Comment:
Do we really want to extract `uri` and send it with `src` name? 
Is this a bug in current master? (Just make use it doesn't break any functionality)
And use  `const { uri } = src;`

Reply:
I tested it and it's working fine on ios and android!
Can you check i have another commit for the Presence type

Reply:
@eobuJ is this an intentional fix or a mistake? If it was a bug that you are fixing, you can separate it in a different commit (one for adding Flow types, one for fixing a bug). But if it is not a fix, then it is a mistake?

Reply:
It's not a fixed. Because I added a type of ImageResource and it has a uri property which is a string, so flow explicitly requires a string as input. It compiles and functions normally by passing the src, instead of accessing the uri. So to avoid getting the errors from flow I just access the uri property and pass it as an input.
Here is the type
export type ImageResource = {
  uri: string,
}; @borisyankov

Reply:
Run `npm run prettier` to fix your code formatting.